[func] mo server instance crashed by SIGSEGV

#### Can be reproduced ?
Yes

#### Steps:
create table tbl1 (col_1a tinyint, col_1b smallint, col_1c int, col_1d bigint, col_1e char(10) not null);
insert into tbl1 values (0,1,1,7,"a");
insert into tbl1 values (0,1,2,8,"b");
insert into tbl1 values (0,1,3,9,"c");
insert into tbl1 values (0,1,4,10,"D");
insert into tbl1 values (0,1,5,11,"a");
insert into tbl1 values (0,1,6,12,"c");
insert into tbl1 values (null,null,null,null,'');
select sum(col_1d)+count(col_1d)+avg(col_1d)+min(col_1d)+max(col_1d)+min(col_1e)+max(col_1e) as sum from tbl1 group by col_1d;

#### Expected behavior:
mo server instance should not crash

#### Actual behavior:
mo server instance crashed by SIGSEGV

#### Logs:
2021/09/15 18:13:36.453065 +0800 INFO time of getDataFromPipeline : 27.458µs
2021/09/15 18:13:36.453071 +0800 INFO time of Exec.Run : 1.53525ms
2021/09/15 18:13:36.453075 +0800 INFO connection id 2 , the time of handling the request 1.921125ms
2021/09/15 18:13:36.453204 +0800 INFO cmd 3
2021/09/15 18:13:36.453210 +0800 INFO query:select sum(col_1d)+count(col_1d)+avg(col_1d)+min(c...
2021/09/15 18:13:36.453545 +0800 INFO time of Exec.Compile : 284.958µs
+++++++++
		:0 π(col_1d,col_1e) -> γ([col_1d], [sum(col_1d) -> sum(col_1d), count(col_1d) -> count(col_1d), sumCount(col_1d) -> avg(col_1d), min(col_1d) -> min(col_1d), max(col_1d) -> max(col_1d), min(col_1e) -> min(col_1e), max(col_1e) -> max(col_1e)] -> =>
	:2 γ([col_1d], [sum(sum(col_1d)) -> sum(col_1d), sum(count(col_1d)) -> count(col_1d), sumCount(avg(col_1d)) -> avg(col_1d), min(min(col_1d)) -> min(col_1d), max(max(col_1d)) -> max(col_1d), min(min(col_1e)) -> min(col_1e), max(max(col_1e)) -> max(col_1e)] -> =>
:1 γ([col_1d], [sum(sum(col_1d)) -> sum(col_1d), sum(count(col_1d)) -> count(col_1d), avg(avg(col_1d)) -> avg(col_1d), min(min(col_1d)) -> min(col_1d), max(max(col_1d)) -> max(col_1d), min(min(col_1e)) -> min(col_1e), max(max(col_1e)) -> max(col_1e)] -> π(sum(col_1d) + count(col_1d) + avg(col_1d) + min(col_1d) + max(col_1d) + min(col_1e) + max(col_1e)) -> sql output
+++++++++
2021/09/15 18:13:36.454020 +0800 INFO session connected {"session-id": 39, "addr": "127.0.0.1:60970"}
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x8 pc=0x102ba93b4]

goroutine 2800 [running]:
github.com/RoaringBitmap/roaring/roaring64.(*Bitmap).Or(0x140007b39f0, 0x0)
	/Users/yangli/go/pkg/mod/github.com/!roaring!bitmap/roaring@v0.9.4/roaring64/roaring64.go:665 +0x34
matrixone/pkg/container/nulls.Or(0x1400202e870, 0x1400202e880, 0x1400202e8d0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/container/nulls/nulls.go:38 +0x12c
matrixone/pkg/sql/colexec/extend/overload.init.13.func4(0x140007b3310, 0x140007b3400, 0x140030192c0, 0x0, 0x0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/overload/plus.go:302 +0x8cc
matrixone/pkg/sql/colexec/extend/overload.BinaryEval(0x4, 0x5, 0x5, 0x0, 0x0, 0x140007b3310, 0x140007b3400, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/overload/binary.go:36 +0x1b0
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0x140023025a0, 0x14001d4f5e0, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/binary.go:53 +0x12c
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0x140023025d0, 0x14001d4f5e0, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/binary.go:45 +0x44
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0x14002302600, 0x14001d4f5e0, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/binary.go:45 +0x44
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0x14002302630, 0x14001d4f5e0, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/binary.go:45 +0x44
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0x14002302660, 0x14001d4f5e0, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/binary.go:45 +0x44
matrixone/pkg/sql/colexec/extend.(*BinaryExtend).Eval(0x14002302690, 0x14001d4f5e0, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/extend/binary.go:45 +0x44
matrixone/pkg/sql/colexec/projection.Call(0x140030192c0, {0x103e1c280, 0x14002fea680})
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/colexec/projection/projection.go:54 +0x1d8
matrixone/pkg/vm.Run({0x140003d2240, 0x3, 0x4}, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/vm/vm.go:49 +0xa4
matrixone/pkg/vm/pipeline.(*Pipeline).RunMerge(0x140035c5f00, 0x140030192c0)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/vm/pipeline/pipeline.go:117 +0x154
matrixone/pkg/sql/compile.(*Scope).MergeRun(0x14002b26500, {0x10415a458, 0x140025a0000})
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/compile/scope.go:116 +0x258
matrixone/pkg/sql/compile.(*Exec).Run.func2(0x14002b26080, 0x1400041fb20, 0x14002b26500)
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/compile/compile.go:161 +0x3c
created by matrixone/pkg/sql/compile.(*Exec).Run
	/Users/yangli/Documents/Origin/workspace/matrixone/pkg/sql/compile/compile.go:160 +0x188

### Environment:
Commit-id : c80b16a7f987d597b9b20d6c438d0df4de591b46
MySQL Client: mysql Ver 8.0.26 for macos11.4 on arm64 (Homebrew)

--- a/pkg/container/nulls/nulls.go
+++ b/pkg/container/nulls/nulls.go
@@ -28,14 +28,14 @@ func Or(n, m, r *Nulls) {
 		if r.Np == nil {
 			r.Np = roaring.NewBitmap()
 		}
-		r.Np.Or(roaring.NewBitmap())
+		r.Np.Or(m.Np)
 		return
 	}
 	if (m == nil || m != nil && m.Np == nil) && n != nil && n.Np != nil {
 		if r.Np == nil {
 			r.Np = roaring.NewBitmap()
 		}
-		r.Np.Or(roaring.NewBitmap())
+		r.Np.Or(n.Np)
 		return
 	}
 	if m != nil && m.Np != nil && n != nil && n.Np != nil {
@@ -46,6 +46,7 @@ func Or(n, m, r *Nulls) {
 		r.Np.Or(m.Np)
 	}
 }
+
 
 func (n *Nulls) Reset() {
 	if n.Np != nil {

